Sources/Tracking/CustomerIO.swift

```diff
@@ -10,6 +10,17 @@ public protocol CustomerIOInstance: AutoMockable {
         onComplete: @escaping (Result<Void, CustomerIOError>) -> Void,
         jsonEncoder: JSONEncoder?
     )
+
+    // sourcery:Name=track
+    func track<RequestBody: Encodable>(
```

Comment:
Should we move `track()` into the `Tracking` class? We talked about `identify` being in `CustomerIO` since all features depend on it but this function shouldn't have that same problem, yeah?

Reply:
I spent some time tooling around with doing the stuff in tracking and I think without a reorg of the `identifyRepository` stuff it ends up being overly complicated. Again referring to our other libraries, what we tend to see as the core / in the box functionality is always identifies & events, so IMO it makes sense for those to all be in the same place, instead of having to wrap a whole other object